pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.akka</groupId>
+    <artifactId>akka-javasdk-parent</artifactId>
+    <version>3.3.0</version>
+  </parent>
+  
+  <groupId>io.github.efgpinto</groupId>
+  <artifactId>akka-rag-example</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>akka-rag-example</name>
+  <properties>
+    <jackson.version>2.17.0</jackson.version>
+    <langchain4j.version>1.0.0-beta2</langchain4j.version>
+
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+        <version>${jackson.version}</version>
```

Comment:
This dependency shouldn't be needed for langchain with open-ai, as far as I know.

Reply:
It is.. but only when using `1.0.0-beta2` .. not with beta1 apparently.